fab/fab/services/templates/supervisor_celery_worker.conf

```diff
@@ -1,16 +1,16 @@
-[program:{{ project }}-{{ environment }}-celery_{{ celery_params.queue }}_{{ celery_params.worker_num }}]
+[program:{{ project }}-{{ environment }}-celery_{{ celery_params.comma_separated_queue_names }}_{{ celery_params.worker_num }}]
 environment={% for name, value in supervisor_env_vars.items %}{{ name }}="{{ value }}"{% if not forloop.last %},{% endif %}{% endfor %}
-command=/bin/bash {{ code_current }}/services/supervisor/{{ environment }}_celery_bash_runner{% if celery_params.optimize %}_optimized{% endif %}.sh --queues={{ celery_params.queue }} --events --loglevel=INFO --hostname={{ host_string }}_{{ celery_params.queue }}_{{ celery_params.worker_num }} {% if celery_params.pooling == 'prefork' %} --autoscale={{ celery_params.concurrency }},0 -Ofair --maxtasksperchild={{ celery_params.max_tasks_per_child }}{% endif %}{% if celery_params.pooling == 'gevent' %} -P gevent --concurrency={{ celery_params.concurrency }}{% endif %} --without-gossip
+command=/bin/bash {{ code_current }}/services/supervisor/{{ environment }}_celery_bash_runner{% if celery_params.optimize %}_optimized{% endif %}.sh --queues={{ celery_params.comma_separated_queue_names }} --events --loglevel=INFO --hostname={{ host_string }}_{{ celery_params.queue }}_{{ celery_params.worker_num }} {% if celery_params.pooling == 'prefork' %} --autoscale={{ celery_params.concurrency }},0 -Ofair --maxtasksperchild={{ celery_params.max_tasks_per_child }}{% endif %}{% if celery_params.pooling == 'gevent' %} -P gevent --concurrency={{ celery_params.concurrency }}{% endif %} --without-gossip
```

Comment:
the hostname will need to be updated as well

Reply:
aha, good catch. I think I missed it in my find-all because it's on the same line.